keycloak/keycloak-admin-ui
# Realm roles: add "users in role" tab

## Motivation


https://issues.redhat.com/browse/APPDUX-843

## Brief Description

Adds "Users in role" tab to realm roles section of Keycloak.

## Verification Steps


1. Go to Realm Roles section.
2. Create a new realm role.
3. Click on the "Users in role tab".
4. Verify that the empty state appears.
5. In the old console, go to Users -> Add user. 
6. Make sure to fill out each of the fields Username, Email, First Name, and Last Name
7. Click save and go to the Role Mappings tab.
8. Select the realm role you created in the "Realm roles" box and click "Add selected".
9. Go back to the admin console and refresh.
10. Select the "Users in Role" tab again and verify that the role information displays in the data table. 

## Checklist:


## Additional Notes
